packages/services/f-analytics/src/plugins/analytics.plugin.js

```diff
@@ -1,8 +1,40 @@
 import analyticsModule from '../store/analytics.module';
 import AnalyticService from './lib/analytics.service';
+import {
+    PUSH_PLATFORM_DATA,
+    PUSH_EVENT
+} from '../store/mutation-types';
 
 const defaults = require('./defaults');
 
+const getCookie = (name, req) => {
+    if (req && req.headers && req.headers.cookie) {
+        const value = `; ${req.headers.cookie}`;
+        const parts = value.split(`; ${name}=`);
+        if (parts.length === 2) {
+            return parts.pop().split(';').shift();
+        }
+    }
+
+    return undefined;
+};
+
+const mapServersideValues = (store, req, options) => {
```

Comment:
We will need to extend this after 0.7.0 to grab the httpStatusCode and dispatch to store pageData

Reply:
Yes, we have a ticket for that